### ISSUE TITLE
use viral-ngs for ion torrent data?

### ISSUE THREAD

**Issue by biocyberman** [broadinstitute/viral-ngs#570]
I am setting up this `viral-ngs` package to try out. However, it would be nice to hear some experiences or opinions about using it for data of human pathogenic virus sequenced by an ion torrent proton. That will surely save me sometimes from trial and error iterations.

**Comment by jrotieno** [comment]
I used the pipeline sometime last year for respiratory syncytial virus assembly, and it worked great. Had a few deletion errors in a few of the assemblies for a region I was quite sure had a duplication. This was easy to correct as we had sanger contigs for that region. Best!

**Comment by dpark01** [comment]
I am also quite curious to hear about how well this package works for Ion Torrent data -- although much of our depletion and assembly code is theoretically platform agnostic, we haven't validated it ourselves on such data.

@jrotieno that's great to hear the RSV assembly worked out. Does KEMRI have an Ion Torrent for pathogen sequencing? I'm also curious to hear if you think that the deletion errors were algorithmic (viral-ngs's fault) or mostly driven by the read data itself.

**Comment by jrotieno** [comment]
@dpark01 We had an Ion Torrent machine but traded it in for a MiSeq. A few people here used it for plasmodium and bacterial 16S sequencing.

I think the deletion errors were algorithmic as didn't find the same errors with Spades for the same samples and reads. However, I had also experienced similar indels problems with SPades, masurca and vicuna.

I settled on viral-ngs as I only had issues on regions I clearly had an idea of if something went wrong! In addition, I quite understand the pipeline!
